src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_command_factory.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SRC_COMPONENTS_APPLICATION_MANAGER_RPC_PLUGINS_RC_RPC_PLUGIN_INCLUDE_RC_RPC_PLUGIN_RC_COMMAND_FACTORY_H
```

Comment:
.../application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_command_factory.h

why there is a second rc_rpc_plugin folder under include?  Shall it be just include, since include is already under rc_rpc_plugin?
This comment applies to all file in the same folder.